lib/board.rb

```diff
@@ -0,0 +1,48 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    cells.clear
```

Comment:
@johnfewell line 10 is not necessary since you are setting @cells to be a new array.